lib/SIL/IR/SILPrinter.cpp

```diff
@@ -3380,7 +3380,7 @@ void SILFunction::print(SILPrintContext &PrintCtx) const {
   case IsReabstractionThunk: OS << "[reabstraction_thunk] "; break;
   }
   if (isDynamicallyReplaceable()) {
-    OS << "[dynamically_replacable] ";
+    OS << "[dynamically_replaceable] ";
```

Comment:
@AnthonyLatsis should revert, right?

Reply:
You can change it, but in this case you have to change all occurrences in all test files, too. If you want to do this, I recommend to put those changes in a separate commit

Reply:
I would extract that change into a separate PR, this one is quite large already. If you choose to revert, please make sure to audit all modified occurrences.

Reply:
I'm leaving it as is.

Reply:
Then please put everything related to `dynamically_replaceable` in a separate commit, and make sure to modify *all* occurrences across the repository. Otherwise the tests will not pass, and we don’t want comments and documentation that mention this compiler attribute to be out-of-date.

Reply:
@AnthonyLatsis I've changed all occurrence to -> `dynamically_replacable` in seperate commit: https://github.com/swiftlang/swift/pull/75020/commits/08ba5a57996f38594baafe3d4f6eb4c593914e02